scrapper/models.go

```diff
@@ -126,23 +126,24 @@ func (r TableRow) TableFqn() DwhFqn {
 }
 
 type SqlDefinitionRow struct {
-	Instance string `db:"instance" json:"instance" ch:"instance" bigquery:"instance"`
-	Database string `db:"database" json:"database" ch:"_database" bigquery:"database"`
-	Schema   string `db:"schema" json:"schema" ch:"schema" bigquery:"schema"`
-	Table    string `db:"table" json:"table" ch:"table" bigquery:"table"`
-	IsView   bool   `db:"is_view" json:"is_view" ch:"is_view" bigquery:"is_view"`
-	Sql      string `db:"sql" json:"sql" ch:"sql" bigquery:"sql"`
+	Instance           string `db:"instance"             json:"instance"             ch:"instance"             bigquery:"instance"`
+	Database           string `db:"database"             json:"database"             ch:"_database"            bigquery:"database"`
+	Schema             string `db:"schema"               json:"schema"               ch:"schema"               bigquery:"schema"`
+	Table              string `db:"table"                json:"table"                ch:"table"                bigquery:"table"`
+	IsView             bool   `db:"is_view"              json:"is_view"              ch:"is_view"              bigquery:"is_view"`
+	IsMaterializedView bool   `db:"is_materialized_view" json:"is_materialized_view" ch:"is_materialized_view" bigquery:"is_materialized_view"`
+	Sql                string `db:"sql"                  json:"sql"                  ch:"sql"                  bigquery:"sql"`
 }
 
 type DatabaseRow struct {
-	Instance      string  `db:"instance" json:"instance" ch:"instance" bigquery:"instance"`
-	Database      string  `db:"database" json:"database" ch:"_database" bigquery:"database"`
-	Description   *string `db:"description" json:"description" ch:"description" bigquery:"description"`
-	DatabaseType  *string `db:"database_type" json:"database_type" ch:"database_type" bigquery:"database_type"`
+	Instance      string  `db:"instance"       json:"instance"       ch:"instance"       bigquery:"instance"`
+	Database      string  `db:"database"       json:"database"       ch:"_database"      bigquery:"database"`
+	Description   *string `db:"description"    json:"description"    ch:"description"    bigquery:"description"`
+	DatabaseType  *string `db:"database_type"  json:"database_type"  ch:"database_type"  bigquery:"database_type"`
 	DatabaseOwner *string `db:"database_owner" json:"database_owner" ch:"database_owner" bigquery:"database_owner"`
 }
 
-func (r DatabaseRow) SetInstance(instance string) {
+func (r *DatabaseRow) SetInstance(instance string) {
```

Comment:
changed receiver to pointer to modify original ... I think it is purpose of this method.